angular/projects/admin/src/app/services/document.service.ts

```diff
@@ -111,15 +118,30 @@ export class DocumentService {
         query = query.orderBy(queryOpts.orderBy.field, queryOpts.orderBy.sortOrder);
       }
 
+      if (!nextPage && backDocSnap) { // back
+        query = query.startAt(backDocSnap);
+      }
+
       if (queryOpts.limit) {
         query = query.limit(queryOpts.limit);
       }
 
+      if (nextPage && lastDocSnap) { // next page
+        query = query.startAfter(lastDocSnap);
+      }
```

Comment:
Remove this `if` statement

Reply:
Resolved on this commit https://github.com/oddbit/tanam/pull/220/commits/4abec6e1632ed1302d8f7aa4287a184718f1352b